### PR TITLE
Update paper-autocomplete.html

### PR DESCRIPTION
**Pull request by aruntk**
paper-input's [allowedPattern](https://elements.polymer-project.org/elements/paper-input#property-allowedPattern) and [pattern](https://elements.polymer-project.org/elements/paper-input#property-pattern) properties added.

### CODE REVIEW

**Comment by tachyon1337** [comment]
I look it over and then merge the request. Thanks!

File: paper-autocomplete.html [hunk 1]
```diff
@@ -65,158 +62,180 @@
...
       noLabelFloat:{
         type: Boolean,
         value: false
-      },
-
-       /**
-       * `required` Set to true to mark the input as required.
-       */
+      }
+      ,
```
Comment (jhuesos) [inline paper-autocomplete.html:122]: I would be careful with formatting the code. The list of changes in this PR is pretty big, but most of the changes are only style changes. I guess the maintainers will have something to say about this.

Comment (tachyon1337) [reply paper-autocomplete.html:122]: @jhuesos 

I would do it manually by hand in this instance

**Comment by tachyon1337** [comment]
I manually updated the repo to support regular expression patterns for input.

**Pull request closed without merging**